Corrected Export Command Placement (#787)
kyma-project/btp-manager: docs/contributor/09-10-ui.md

> [!WARNING]
> This feature is in the experimental stage and is not yet available in the main branch or official releases.

> 

Author: That would be out of scope of this PR. Let's handle it in other PRs.

## Prerequisites
1. Connect `kubectl` to your cluster by setting the **KUBECONFIG** environment variable.
    ```shell
    export KUBECONFIG=<path-to-kubeconfig>
    ```

2. For clusters different from Kyma (for example, k3d) you need to install the [prerequisites](../../deployments/prerequisites.yaml).
    ```shell
    kubectl apply -f deployments/prerequisites.yaml
    ```

## Run BTP Manager with UI
Follow the steps below to run BTP Manager with UI:

> 1. Make sure the `btp-operator` module is disabled and there are no existing BtpOperator custom resources (CRs) and deployments of BTP Manager and the SAP BTP service operator.

Reviewer: ```suggestion
1. Ensure that the SAP BTP Operator module is deleted and verify that there are no existing BtpOperator custom resources (CRs) or deployments of both BTP Manager and the SAP BTP service operator.
```

    ```shell
    kubectl get btpoperators -A
    kubectl get deployment -n kyma-system btp-manager-controller-manager
    kubectl get deployment -n kyma-system sap-btp-operator-controller-manager
    ```

> 2. Clone the `btp-manager` repository and checkout to the `sm-integration` branch.

Reviewer: ```suggestion
2. Clone the `btp-manager` repository and check out to the `sm-integration` branch.
```

    ```shell
    git clone https://github.com/kyma-project/btp-manager.git
    git checkout sm-integration
    ```
3. Set the **IMG** environment variable to the image of BTP Manager with UI.
    ```shell
    export IMG=europe-docker.pkg.dev/kyma-project/dev/btp-manager:PR-779
    ```
4. Run `deploy` makefile rule to deploy BTP Manager with UI.
    ```shell
    make deploy
    ```
5. Check if BTP Manager deployment is running.
    ```shell
    kubectl get deployment -n kyma-system btp-manager-controller-manager
    ```
    If you encounter the following error during Pod creation due to Warden's admission webhook:
    ```
    Error creating: admission webhook "validation.webhook.warden.kyma-project.io" denied the request: Pod images europe-docker.pkg.dev/kyma-project/dev/btp-manager:PR-779 validation failed
    ```

>     you must scale the BTP Manager deployment to 0 replicas, delete the webhook, and the scale the deployment back to 1 replica.

Reviewer: ```suggestion
    you must scale the BTP Manager deployment to 0 replicas, delete the webhook, and then scale the deployment back to 1 replica.
```
